packages/dds/merge-tree/src/localReference.ts

```diff
@@ -101,6 +101,10 @@ export class LocalReference implements ReferencePosition {
         this.properties = addProperties(this.properties, newProps, op);
     }
 
+    public getClient() {
```

Comment:
i don't love exposing this, as i've been trying to keep it hidden on sequence, but need to see how it's used further in the PR.